spec.md

```diff
@@ -471,6 +471,7 @@ how Platforms might expose these values to their users.
 | metadata | object | An opaque object of metadata for a Service Plan. It is expected that Platforms will treat this as a blob. Note that there are [conventions](profile.md#service-metadata) in existing Service Brokers and Platforms for fields that aid in the display of catalog data. |
 | free | boolean | When false, Service Instances of this plan have a cost. The default is true. |
 | bindable | boolean | Specifies whether Service Instances of the Service Plan can be bound to applications. This field is OPTIONAL. If specified, this takes precedence over the `bindable` attribute of the service. If not specified, the default is derived from the service. |
+| updateable | boolean | Whether the service supports upgrade/downgrade for some Service Plans. Please note that the attribute is intentionally misspelled as `updateable` for legacy reasons. Defaults to false. |
```

Comment:
Two things:
1. Is it really upgrade/downgrade, or is it 'different'?
2. Is matching an existing fields spelling for legacy? That seems strange, but also correct to me.

Reply:
Thinking more about this.... I think the "Defaults to false" might need to be something like "If not specified then the service's 'updateable' field applies". Since we don't want this to change the current behavior if its not present.